automation_tools/satellite6/capsule.py

```diff
@@ -225,7 +217,8 @@ def register_capsule():
     """Register the capsule on the Satellite 6 server."""
     config = _get_config()
     run(
-        'rpm -Uvh http://{}/pub/katello-ca-consumer-latest.noarch.rpm'
+        'yum -y localinstall '
+        'http://{}/pub/katello-ca-consumer-latest.noarch.rpm'
```

Comment:
localinstall have the same effect as rpm -Uvh. When  passing an URL it's gonna first download the RPM and then install it locally